extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -289,7 +294,7 @@ export class ImageViewer {
     // and then naturally upgrade to a higher quality image.
     return timerFor(this.win).promise(1).then(() => {
       this.image_.setAttribute('src', src);
-      return this.loadPromise(this.image_);
```

Comment:
I see thanks for explaining. Should we then bind the `this.loadPromise.bind(this)` and pass it to make sure the context is transferred. 

I am cool with this either way.